include/mata/nft/nft.hh

```diff
@@ -62,24 +62,22 @@ public:
 
 public:
     explicit Nft(Delta delta = {}, utils::SparseSet<State> initial_states = {},
-                 utils::SparseSet<State> final_states = {}, std::vector<Level> levels = {}, const Level levels_cnt = 1,
```

Comment:
Thinking about it, should it not be 2 by default? 0 is invalid state, 1 is NFA, 2 is the most basic NFT.

Reply:
I'm going to change the default value to 1, which indicates a similarity to an NFA structure. This is primarily because an NFT constructor can be invoked without the need to specify levels. If no levels are specified (i.e., level_cnt == 1), the automaton will be handled as an NFA.

Reply:
This is up to debate, I think. I would expect by default to use two levels, since you are creating an NFT explicitly, not an NFA. In general, I think that `Nft nft{}` should create the most typical version of NFT, that is, NFT that has two levels, in my opinion.

Reply:
It's really hard to determine the best approach. The value 1 for levels_cnt has been chosen to represent the simplest possible NFT (equivalent to NFA). When you create Nft nft{}, you don't know its levels, so I think it would be dangerous to assume that levels_cnt == 2 as it could affect other algorithms (like get_one_level_aut).

I would leave it as is for now.

Reply:
Agreed. I made a note to open this discussion again.